assets/helpers/flashSale.js

```diff
@@ -6,13 +6,16 @@ import { getProductPrice } from 'helpers/subscriptions';
 
 import type { SubscriptionProduct } from './subscriptions';
 
+// Get end time
+function getEndTime(): number { return new Date(2018, 10, 5, 0, 0).getTime();; }
```

Comment:
Can you use the same date as is used in the flashSaleIsActive() function above? Then we'll only need to change the date in one place next time we do this.

Reply:
Great point! all set